docs/concepts/jcloud/yaml-spec.md

```diff
@@ -69,83 +81,27 @@ If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G mem
 When using GPU resources, it may take a few extra minutes before all Executors are ready to serve traffic.
 ```
 
-#### Shared GPU
+| Instance | GPU    | Memory | Credit Per Hour |
+|----------|--------|--------|-----------------|
+| G1       | shared | 14G    | 100             |
+| G2       | 1      | 14G    | 125             |
+| G3       | 2      | 24G    | 250             |
+| G4       | 4      | 56G    | 500             |
+
+##### Shared GPU
 
 An Executor using a `shared` GPU shares this GPU with up to four other Executors.
 This enables time-slicing, which allows workloads that land on oversubscribed GPUs to interleave with one another.
 
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        gpu: shared
-```
-
-```{caution}
 The tradeoffs with a `shared` GPU are increased latency, jitter, and potential out-of-memory (OOM) conditions when many different applications are time-slicing on the GPU. If your application is memory consuming, we suggest using a dedicated GPU.
-```
 
-#### Dedicated GPU
+To use `shared` GPU, `G1` needs to be specificied as Instance Type.
 
-Using a dedicated GPU is the default way to provision GPU for an Executor. This automatically creates nodes or assigns the Executor to land on a GPU node. In this case, the Executor owns the whole GPU. You can assign between 1 and 4 GPUs.
+##### Dedicated GPU
 
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        gpu: 1
-```
-
-### Spot vs on-demand instance
-
-For cost optimization, JCloud tries to deploy all Executors on `spot` capacity. This is ideal for stateless Executors, which can withstand interruptions and restarts. It is recommended to use `on-demand` capacity for stateful Executors (e.g. indexers) however.
-
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        capacity: on-demand
-```
-
-### Memory
-
-By default, `100M` of RAM is allocated to each Executor. You can use the `memory` argument under `resources` to change it.
-
-```{hint}
-Maximum of 16G RAM is allowed per Executor.
-```
-
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        memory: 8G
-```
+Using a dedicated GPU is the default way to provision GPU for an Executor. This automatically creates nodes or assigns the Executor to land on a GPU node. In this case, the Executor owns the whole GPU. 
```

Comment:
```suggestion
Using a dedicated GPU is the default way to provision a GPU for an Executor. This automatically creates nodes or assigns the Executor to land on a GPU node. In this case, the Executor owns the whole GPU.
```

What do you mean by `land`?

Reply:
I'm not exactly sure, let me check with the team.

Reply:
there's nothing special to it, maybe we can just do:

```
assigns the Executor to a GPU node
```

?

Reply:
Yup, that sounds better

Reply:
fixed